assets/scripts/modules/Copyable.js

```diff
@@ -0,0 +1,82 @@
+import { module } from 'modujs'
+
+export default class extends module {
```

Comment:
I suggest including a block comment with examples of how to use this module.

```suggestion
/**
 * Module: NAME
 *
 * Example of element copying inner HTML:
 *
 * ```html
 * ...
 * ```
 *
 * Example of element copying text from HTML attribute:
 *
 * ```html
 * ...
 * ```
 */
export default class extends module {
```